### PR TITLE
removed named-consumer plus other various fixes

### PR DESCRIPTION
**Pull request by siamakh**

### CODE REVIEW

**Review by rogerkibbe — CHANGES_REQUESTED** [review]
Need to fix capsule.bxb merge - Dylan did updates yesterday to the version #'s to remove deprecation warnings

**Review by rogerkibbe — APPROVED** [review]
Ignore the last - precoffeee

File: README.md [hunk 1]
```diff
@@ -160,12 +160,12 @@ queries and inner/outer property projections.
...
 You can see all trained utterances and plans by entering this query in the
-training tab search bar: `goal:Order#commitOrder -has:continue`. For example:
+training tab search bar: `goal:CommitOrder -has:continue`. For example:
 
 - Make a reservation for a space resort on Mars the third weekend in December
 for 2 astronauts
 
-We train these to the goal `Order#commitOrder` where `commitOrder` is a
+We train these to the goal `CommitOrder` where `commitOrder` is a
 named-consumer on the `Order`. We also add two flagged signal routes:
```
Comment (sarahmolinerroy) [inline README.md:169]: Remove "where `commitOrder` is a
named-consumer on the `Order`."

Comment (sarahmolinerroy) [reply README.md:169]: I'll update it in my PR

**Pull request merged**